source/localizable/models/customizing-serializers.md

```diff
@@ -708,13 +738,38 @@ export default DS.Model.extend({
 }
 ```
 
-Every serialized record must follow this format for it to be correctly
-converted into an Ember Data record.
+Note that the type is `"post"` to match the post model and the
+`relatedPosts` relationship in the document matches the
+`relatedPosts: hasMany('post')` on the model.
+
+#### Normalizing adapter responses
+
+When creating a custom serializer you will need define a
+[normalizeResponse](http://emberjs.com/api/data/classes/DS.Serializer.html#method_normalizeResponse)
+method to transform the response from the adapter into the normalized
+JSON object described above. This method receives the `store`, the
+Model class for the request, the payload payload, the id of the record
+request (or null if there is no id associated with the request) and
+the request type (a string with the possible values of: `'findRecord'`,
+`'queryRecord'`, `'findAll'`, `'findBelongsTo'`, `'findHasMany'`,
+`'findMany'`, `'query'`, `'createRecord'`, `'deleteRecord'` and
+`'updateRecord'`) as arguments.
+
+A custom serializer will also need to define a
+[normalize](http://emberjs.com/api/data/classes/DS.Serializer.html#method_normalize)
+method. This method is called by `store.normalize(type, payload)` and
+is often used for normalizing requests made outside of Ember Data
+because they do not fall into the normal CRUD flow that the adapter
+provides.
+
+#### Serializing records
+
+Finally a serializer will need to implement a
+[serialize](http://emberjs.com/api/data/classes/DS.Serializer.html#method_serialize)
+method. Ember Data will provide a record snapshot and an options hash
+and this method should return an object that the adapter will send to
+the server when creating, updating or deleting a record.
```

Comment:
Should this be more adapter implementation agnostic and say something like: "… the the adapter will receive for `createRecord`, `updateRecord` or `deleteRecord`"?